<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListeningDialog</class>
 <widget class="QDialog" name="ListeningDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>131</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Listening</string>
  </property>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QLabel" name="stateLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Not listening</string>
   </property>
  </widget>
  <widget class="QPushButton" name="helpButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Help...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="configureButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Configure...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="listenButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Start listening</string>
   </property>
  </widget>
  <widget class="QLabel" name="addressLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>No address configured!</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
